change/@ni-nimble-components-d62c8a19-26e8-448d-831b-ddc56969c9bb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
In playing with the [wafer map documentation storybook in the PR build](https://60e89457a987cf003efc0a5b-cqiwtvudij.chromatic.com/?path=/story/wafermap--wafer-map&args=colorScale:set1;highlightedValues:set4;maxCharacters:10;orientation:top&globals=backgrounds.value:!hex(F4F4F4)) I noticed the following:
1. `color-scale-mode` doesn't seem to have any effect
2. many of the values for `highlightedValues` produce the same outcome

I'm not sure how much of this is expected at this stage. Feel free to defer this if there is more work planned, but eventually I would expect each option to produce a noticeable change in appearance.

Reply:
1. We have to investigate it - I'd suggest to defer it and make it our next priority to fix it.
2. I fixed this. The array which provides the sets had the wrong values. Now scenario 2,3,4 should have different values highlighted. Scenario 1 should not have anything highlighted.